README.md

```diff
@@ -2,43 +2,75 @@
 
 [![License](https://img.shields.io/github/license/KIT-MRT/arbitration_graphs)](./LICENSE)
 
-Arbitration graphs combine simple atomic behavior components into more complex behaviors for decision-making and behavior generation
+**Hierarchical behavior models for complex decision-making and behavior generation in robotics!**
 
-## Demonstration
+<!-- 
+TODO: add example arbitration graph from robotics (with POMDP, RL methods)
+<img
+  src="./docs/assets/img/pacman_arbitrator_safe.svg"
+  alt="Example arbitration graph from robotics"
+  style="margin:10px; width: 50%"
+  align="right"
+/>
+-->
 
-We are currently working on a demonstration of this library using Pac-Man as an example application.
+- **Bottom-up** 🌱: Combine simple atomic behavior components to generate complex behaviors.
+- **Functional decomposition** 🧩: Behavior components define the *How to do it?* and *Can we do it?*, while Arbitrators take the decision on *What to do?*
+- **Meta-framework** 🧠: Integrate diverse methods in one decision-making framework. Why not combine optimization-based planning, probabilistic approaches (POMDPs), and machine learning (RL)? Use any approach where it performs best!
+- **Scalability** 📈: Stack behavior components in arbitrators to create hierarchical behavior models.
+- **Maintainability** 🛠️: Add new behaviors without having to touch others – did we mention strict modularity and functional decomposition?
+- **Transparency** 💡: Easily follow and understand the decision-making process.
+- **Behavior Verification** 🛡️: Use tightly integrated verifiers to ensure only valid and safe behavior commands are executed.
+- **Graceful Degradation** 🪂: Your behavior is unreliable or unsafe? Arbitrators will gracefully fall back to the next-best option.
```

Comment:
That's a very nice list, I don't think I have anything to add. I'd personally prefer the list with line breaks you posted in the PR but I'm totally fine with both.
```suggestion
- **Bottom-up** 🌱  
  Combine simple atomic behavior components to generate complex behaviors.
- **Functional decomposition** 🧩  
  Behavior components define the *How to do it?* and *Can we do it?*, while Arbitrators take the decision on *What to do?*
- **Meta-framework** 🧠  
  Integrate diverse methods in one decision-making framework. Why not combine optimization-based planning, probabilistic approaches (POMDPs), and machine learning (RL)? Use any approach where it performs best!
- **Scalability** 📈  
  Stack behavior components in arbitrators to create hierarchical behavior models.
- **Maintainability** 🛠️  
  Add new behaviors without having to touch others – did we mention strict modularity and functional decomposition?
- **Transparency** 💡  
  Easily follow and understand the decision-making process.
- **Behavior Verification** 🛡️  
  Use tightly integrated verifiers to ensure only valid and safe behavior commands are executed.
- **Graceful Degradation** 🪂  
  Your behavior is unreliable or unsafe? Arbitrators will gracefully fall back to the next-best option.
```
Also, I like your choice of icons. We could think about moving them to the front of each point but having taken a look at it I think I prefer the version above.

```suggestion
- 🌱 **Bottom-up** 
  Combine simple atomic behavior components to generate complex behaviors.
- 🧩 **Functional decomposition**
  Behavior components define the *How to do it?* and *Can we do it?*, while Arbitrators take the decision on *What to do?*
 - 🧠 **Meta-framework**
  Integrate diverse methods in one decision-making framework. Why not combine optimization-based planning, probabilistic approaches (POMDPs), and machine learning (RL)? Use any approach where it performs best!
- 📈 **Scalability**
  Stack behavior components in arbitrators to create hierarchical behavior models.
- 🛠️ **Maintainability**
  Add new behaviors without having to touch others – did we mention strict modularity and functional decomposition?
- 💡 **Transparency**
  Easily follow and understand the decision-making process.
- 🛡️ **Behavior Verification**
  Use tightly integrated verifiers to ensure only valid and safe behavior commands are executed.
- 🪂 **Graceful Degradation**
  Your behavior is unreliable or unsafe? Arbitrators will gracefully fall back to the next-best option.
```